frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -217,7 +218,7 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                     session,
                     codeRequest,
                     userContext,
-                    userProfileMaybe.isPresent() ? userProfileMaybe.get().getSubjectID() : null,
+                    userProfile != null ? userProfile.getSubjectID() : null,
```

Comment:
If the journey is reauth can we get to this point with a null subjectId?  Should we add this check to the guard at 161?